components/usage-api/usage/v1/billing.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package usage.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/usage-api/v1";
+
+import "google/protobuf/timestamp.proto";
+import "usage/v1/usage.proto";
+
+service BillingService {
+  rpc UpdateInvoices(UpdateInvoicesRequest) returns (UpdateInvoicesResponse) {};
+}
+
+message UpdateInvoicesRequest {
+  google.protobuf.Timestamp start_time = 1;
+  google.protobuf.Timestamp end_time = 2;
+
+  repeated BilledSession sessions = 3;
```

Comment:
As a client of this RPC I would assume I can:

* Give a start time and an end time and have invoices updated for all instances that ran in that time (maybe I'd like to give attributionIds too, so that I only update invoices for that team/user).

but here I can pass a list of `BilledSessions` as inputs (presumably intended to be those returned by the `ListBilledUsage` RPC on the usage service. What happens if I pass billed sessions that lie outside the time range?

Reply:
So this is not a general purpose RPC, in a way, it's an internal RPC. I can clarify in comments. 

You're correct, this only makes sense in the context of another thing which produces the sessions. I'm not sure how to rename or better convey this but the intended usage is not outside callers without context, but the "usage reconcile run"

Reply:
I think some comments above the service to that effect would be good 👍

Reply:
When we do have authN in depth, I would've also encoded that the only caller of this RPC can be a service account which has the perms to generate usage reports, which I believe would've helped.

Reply:
Added a comment on the proto